api/users/user-models.js

```diff
@@ -11,6 +11,7 @@ const pointOption = {
 };
 
 const userSchema = new Schema({
+  name: String,
   firstName: String,
   lastName: String,
```

Comment:
I have kept them for now to prevent any errors as the `firstName` and `lastName` fields are refactored. Removing the personal info fields was not my focus for this iteration.